templates/pcw.ini

```
@@ -17,26 +17,39 @@ smtp-port = 25
 to = <your_email@suse.com>
 from = pcw@publiccloud.qa.suse.de
 
+
+[cluster.notify]
+# List of people who will be notfied about found EKS clusters
+to = <your_email@suse.com>
+
 [notify.namespace.qac]
 # list of recipients which will be notified in case of issues in certain namespace
 to = <your_email@suse.com>
 
 [cleanup]
-# Specify how many images per flavor get kept
-max-images-per-flavor = 2
-# Max age of an image file
-max-images-age-hours = 24
+# Max age of data storage resources ( used in EC2 only )
+ec2-max-age-days = 1
```

Perhaps not for this PR, but why not define ONE value in `dd:hh:mm` instead of scattering the allowed timestamp over multiple values?

e.g.

```suggestion
ec2-max-age = 1d
```

---

https://github.com/SUSE/pcw/issues/174